ansible/apigateway.yml

```diff
@@ -8,3 +8,5 @@
 - hosts: apigateway
   roles:
   - apigateway
+
+- import_playbook: routemgmt.yml
```

Comment:
in environments where a non-local api gw is used, these actions will still need to be separately installed.